packages/gestalt-datepicker/src/DatePicker.jsdom.test.js

```diff
@@ -5,10 +5,17 @@ import DatePicker from './DatePicker.js';
 
 const initialDate = new Date(2018, 11, 14);
 
-function DatePickerWrap() {
+function DatePickerWrap(showMonthYearDropdown) {
```

Comment:
This shouldn't be working — you're passing this as a prop below, so you need to destructure it. As written, this is the equivalent of
```js
function DatePickerWrap(props) {
  return (
    <DatePicker
      showMonthYearDropdown={!!props}
    />
  );
}
```